base/libgit2/libgit2.jl

```diff
@@ -42,22 +42,37 @@ immutable State
     work::GitHash
 end
 
-"""Return HEAD GitHash as string"""
+"""
+    head(pkg::AbstractString) -> String
```

Comment:
strange for this to be using `pkg` as an argument name given it's more general than that

(though most places that use `repo` do it for a `::GitRepo` input)